osv/ecosystems/debian.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Debian ecosystem helper."""
+
+import json
+import logging
+import requests
+
+from ..third_party.univers.debian import Version as DebianVersion
+
+from . import config
+from .helper_base import Ecosystem, EnumerateError
+from .. import cache
+from ..request_helper import RequestError, RequestHelper
+
+CLOUD_API_CACHE_URL_TEMPLATE = (
+    'https://storage.googleapis.com/debian-osv/first_package_output/'
+    '{version}.json')
+CACHE_DURATION_SECONDS = 60 * 60 * 24
+TIMEOUT = 30  # Timeout for HTTP(S) requests
```

Comment:
Let's just use config.timeout instead?

Reply:
Huh, I didn't notice that - sure.
I didn't introduce that duplication, I'm guessing it was added to avoid circular imports.